content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst

```diff
@@ -8,7 +8,9 @@ Link third-party shipping carriers to Odoo databases to verify carriers' deliver
 addresses, :doc:`automatically calculate shipping costs <delivery_method>`, and :doc:`generate
 shipping labels <../advanced_operations_shipping/labels>`.
 
-In Odoo, shipping carriers can be applied to sales orders (SO), invoices, or delivery orders.
+In Odoo, shipping carriers can be applied to sales orders (SO), invoices, or delivery orders. For
```

Comment:
```suggestion
In Odoo, shipping carriers can be applied to sales orders (SOs), invoices, or delivery orders. For
```
If it's multiple sales orders, should we add an -s here? I know it looks a bit weird so it's up to you